rpcs3/Emu/RSX/VK/vkutils/shared.cpp

```diff
@@ -99,7 +99,8 @@ namespace vk
 		{
 		default:
 		case 0:
-			fmt::throw_exception("Assertion Failed! Vulkan API call failed with unrecoverable error: %s%s", error_message, src_loc{line, col, file, func});
+			if (!message.empty()) message += "\n\n";
+			fmt::throw_exception("%sAssertion Failed! Vulkan API call failed with unrecoverable error: %s%s", message, error_message, src_loc{line, col, file, func});
```

Comment:
Ftr, for some people like myself the fmt error is printed on the commandline and never shows a messagebox. This is going to be doubly true when the exception is raised in the routines that initialize the game window for some reason, but I may be the exception, not the rule. Still something to keep in mind.

Reply:
Maybe try to find out why it doesn't work for you then.
Should be easy enough to debug report_fatal_error function.